### PR TITLE
spicy pickled models

### PR DESCRIPTION
**Pull request by rcehemann**
image

Two new sentiment models using the scikit/learn API.

* added a `SciKit` subclass of `SenPy` which infers the model file from `data['models']` in the request
* new models are accessed by `data['models']` =  `'scikit/lr'` or `'scikit/nb'`
* restructured the model/subclass folders to that imports are more sensible. app.models contains the subclass files, so imports are now
`from .models.mock import Mock`
instead of 
`from .models.mock.mock import Mock`

Changed the readme from 'mock' example to 'spacy' example and added a list of supported models.

Example request with new model:
image

Also merged the `feature/make-dev-server` branch into this to test compatibility. :+1:

### CODE REVIEW

File: app/models/spaCy/spacy.py [hunk 1]
```diff
@@ -6,8 +6,8 @@ def __init__(self):
     return None
 
   def load(self):
-    self.nlp = spacy.load("./app/models/spaCy")
+    self._nlp = spacy.load("./app/models/spaCy")
```
Comment (thekeele) [inline app/models/spaCy/spacy.py:9]: What's the difference between `self.nlp` and `self._nlp`?

File: app/models/spacy.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from app.sen_py import SenPy
+import spacy
+
+class SpaCy(SenPy):
```
Comment (thekeele) [inline app/models/spacy.py:4]: It looks like you have two `SpaCy` models in this PR? Copy/paste error?

File: Makefile [hunk 3]
```diff
@@ -1,12 +1,15 @@
+export FLASK_APP=app
```
Comment (thekeele) [inline Makefile:1]: If you merge `master` I think these changes will go away for this PR.

File: app/models/spaCy/spacy.py (hunk 1 above)
Comment (rcehemann) [reply app/models/spaCy/spacy.py:9]: the leading underscore is just naming convention that says "this is a private variable." There aren't actually public/private variables in python, so it doesn't make a real difference. I'm just trying to be PEP friendly

File: app/models/spacy.py (hunk 2 above)
Comment (rcehemann) [reply app/models/spacy.py:4]: the import statement here gets the actual spaCy library that our best model runs on. Do i import it elsewhere too? I'll check later

File: Makefile (hunk 3 above)
Comment (rcehemann) [reply Makefile:1]: Will do before final merge

**Review by thekeele — APPROVED** [review]
🚀 🍻

**Pull request merged**